spec/config/initializers/csv_types_spec.rb

```diff
@@ -51,9 +17,13 @@
     end
   end
 
-  describe 'no_prod_names' do
-    it 'contains tables' do
-      expect(CSV_TYPES_NO_PROD_NAMES).to eq(NO_PROD_TABLES)
+  describe 'fields checks' do
+    CSV_TYPES_TABLES.each do |upload|
+      it "#{klass_name(upload)} csv type config has_api? is a boolean" do
+        if upload.respond_to?(:has_api?)
```

Comment:
this line is violating a rubocop `Style/IfUnlessModifier: Favor modifier if usage when having a single-line body.`

Reply:
fixed